### DISCUSSION TITLE
APPEARENCE: INSERTION CURSOR UNNOTICEABLE

### DISCUSSION THREAD

**Discussion by leinaxd** [community/community#124060] [Accessibility] [answered]
### TL; DR
INSERTION CURSOR UNNOTICEABLE
- make the cursor white please

### Select Topic Area
Show & Tell
### Body

[ENVIRONMENT]
**High Contrast Theme**

[STEPS]
1. Edit any markdown file 'readme.md' 
2. write some text
3. **place the cursor anywhere**
4. I find hard to notice the Insertion Cursor (that pipe | thing to insert text)

[USER CASE]
**Where is the cursor,** i wonder everytime... i struggle a lot

[SOLUTION]
i. please make it **'white'** not **gray** for 'high contrast' theme. 
ii. maybe Slower flickering
iii. [optional customization] Bigger pipe, or bolder

Thx you very much :)

**Comment by cehfisher** [comment]
Hello @leinaxd - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by ericwbailey** [comment, marked as answer]
Hi @leinaxd, I'm a member of the GitHub Accessibility team and I've been assigned to help out with this issue. 

Unfortunately, we're unable to adjust the cursor without endangering modifications other people may have made to the cursors on their operating systems. Modifying your operating system cursor to change its size, color, and appearance is something you can also do, if that is helpful. Here are some links for how to do this on Windows and MacOS:

- [How to Change Mouse Pointer Size and Style in Windows 11 - How-To-Geek](https://www.howtogeek.com/750995/how-to-change-mouse-pointer-size-and-style-in-windows-11/)
- [How to Change the Mouse Pointer Color and Size on Windows 10](https://www.howtogeek.com/427263/how-to-change-the-mouse-pointer-color-and-size-on-windows-10/)
- [Make the pointer easier to see on Mac - Apple Support](https://support.apple.com/guide/mac-help/make-the-pointer-easier-to-see-mchlp2920/mac)

I also wanted to communicate that we are tracking, and paying attention to the issues you and others are filing, and that feedback plays a crucial role in shaping GitHub’s development. So, thank you for vocalizing your issue and sharing it with it. This information, along with other people's feedback will help make things better.
